Marlin/src/lcd/extui/lib/mks_ui/pic_manager.cpp

```diff
@@ -36,7 +36,7 @@
 #include "../../../../MarlinCore.h"
 
 extern uint16_t DeviceCode;
-extern unsigned char bmp_public_buf[17 * 1024];
+extern unsigned char bmp_public_buf[14 * 1024];
```

Comment:
More 14s :)

My goal is to be able to read code without having to stop to ask "What is that?". I'm probably getting annoying, but if I have to stop to ask I leave a comment 😁

Reply:
They started with 17 * 1024.... probably with the new code, it run out of run... so they set it to a smaller value. This buffer is the display buffer for LVGL. It's good to be as big as we can set.